charts/lakekeeper/Chart.yaml

```
@@ -1,11 +1,11 @@
 apiVersion: v2
 name: lakekeeper
 description: Helm Chart for Lakekeeper - a rust native Iceberg Rest Catalog
-version: 0.1.5
-appVersion: 0.4.3
+version: 0.2.0
+appVersion: 0.5.0
 type: application
-home: https://github.com/hansetag/iceberg-catalog
-icon: https://hansetag.com/landing/logo-BLQxxCCG.png
+home: https://github.com/lakekeeper/lakekeeper
+icon: https://github.com/lakekeeper/lakekeeper/blob/833880b99ff220a98a89769ddf7be2bce807ae76/site/docs/assets/bear.svg
```

I've never used this field, but the current link points to an HTML page, the suggested one to the svg directly.
It looks that one could also specify the base64 of the image here
```suggestion
icon: https://raw.githubusercontent.com/lakekeeper/lakekeeper/refs/heads/main/site/docs/assets/bear.svg
```

---

Awesome, thanks. I wasn't aware base64 was possible. Will keep it in mind!